### PR TITLE
Feature/pelagos 4240 add sort filter to the main query

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: config/packages/fos_elastica.yaml [hunk 1]
```diff
@@ -139,6 +139,10 @@ fos_elastica:
                             properties:
                                 id:
                                     type: 'keyword'
+                        sortingDateForDisplay:
```
Comment (praneethpr) [inline config/packages/fos_elastica.yaml:142]: Couldn't think of a better name for this variable, let me know if you guys think of something, will change it

Comment (mickel1138) [reply config/packages/fos_elastica.yaml:142]: Maybe it's time to bring ApprovalDate to the Dataset entity?

Comment (praneethpr) [reply config/packages/fos_elastica.yaml:142]: What do you mean by ApprovalDate?

Comment (mickel1138) [reply config/packages/fos_elastica.yaml:142]: I just wanted to add "some" date up to the dataset level, but it doesn't seem pratical.

**Review by mickel1138 — APPROVED** [review]
Works. Good change.

**Comment by fightingtexasaggie** [comment]
Per the ticket, should all the accepted datasets group first? I know there was some discussion about this in the ticket, but I'm not  sure it was fully resolved.

**Comment by praneethpr** [comment]
> Per the ticket, should all the accepted datasets group first? I know there was some discussion about this in the ticket, but I'm not sure it was fully resolved.

No grouping of datasets, just display based on sort order

**Pull request merged**